src/decrypt.rs

```diff
@@ -8,63 +8,42 @@
 
 use crate::{encryption, get_pad_key_and_iv, xor, EncryptedChunk, Error, Result};
 use bytes::Bytes;
-use itertools::Itertools;
-use rayon::prelude::*;
 use std::io::Cursor;
 use xor_name::XorName;
 
 pub fn decrypt(src_hashes: Vec<XorName>, encrypted_chunks: &[&EncryptedChunk]) -> Result<Bytes> {
```

Comment:
better rename `encrypted_chunks` to `sorted_encrypted_chunks`, or put comment on top of the function to highlight the input shall be `sorted`.
to avoid future misuse.